src/main/java/command/AddCommand.java

```diff
@@ -0,0 +1,94 @@
+package command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.Todo;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import static java.lang.String.format;
+
+/**
+ * Command to add task into data.
+ *
+ */
+public class AddCommand extends Command {
```

Comment:
I like how the the commands handling addition of a `Todo`, `Deadline` or `Event` has been abstracted into a more generic `AddCommand`. Nice!